.github/workflows/tests.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           DATE="$(date --utc '+%Y%m%d')"
 
           # i.e: deps-ubuntu-24.04-amd64-${DATE}
-          echo "KEY=deps-${ID}-${VERSION_ID}-${ARCH}-${DATE}" >> $GITHUB_OUTPUT
+          echo "KEY=deps-${ID}-${VERSION_ID}-${ARCH}-${DATE}-test-dqlite" >> $GITHUB_OUTPUT
```

Comment:
@marco6 this key is cached on successful run of the Code tests, so if you do make changes to your branch be sure to change this key again.